spec/features/course/discussion/topic_management_spec.rb

```diff
@@ -27,6 +27,24 @@
         expect(page).
           to have_selector('div', text: code_annotation.file.answer.question.assessment.title)
       end
+
+      scenario 'I can reply to a comment topic', js: true do
+        # Randomly create a topic, either code_annotation or answer_comment.
+        topic = [true, false].sample ? code_annotation : answer_comment
+        visit course_topics_path(course)
+
+        comment = 'GOOD WORK!'
+        within find('.post-form') do
+          fill_in 'discussion_post[text]', with: comment
+          click_button 'course.discussion.posts.form.comment'
+        end
+        wait_for_ajax
+
+        post = topic.posts.reload.last
+        expect(post.text).to have_tag('*', text: comment)
```

Comment:
It doesn't work because comment is not pure text ( it's the text from HTML editor ).

There are similar expectations in some other places like: https://github.com/Coursemology/coursemology2/blob/master/spec/features/course/assessment/submission/programming_answer_comment_spec.rb#L81